seacatauth/openidconnect/handler/introspect.py

```diff
@@ -42,15 +42,31 @@ def __init__(self, app, oidc_svc, credentials_svc):
 
 	async def introspect(self, request):
 		"""
-		OAuth 2.0 Access Token Introspection Endpoint
-
-		RFC7662 chapter 2
-
-		POST /introspect HTTP/1.1
-		Accept: application/json
-		Content-Type: application/x-www-form-urlencoded
+		OAuth 2.0 Token Introspection Endpoint
+		https://datatracker.ietf.org/doc/html/rfc7662#section-2
+		OAuth 2.0 endpoint that takes a parameter representing an OAuth 2.0 token and returns a JSON document
+		representing the meta information surrounding the token, including whether this token is currently active.
+
+		To protect this endpoint with authorization (as required by RFC7662), use NGINX reverse proxy
+		with auth_request to an NGINX introspection endpoint, for example:
```

Comment:
I guess this need more documentation. So as a seacat user:

1) what endpoints should I call?
2) where do I get the credentials?
3) are those credentials different per each oauth client?

Reply:
Clarified on call